### PR TITLE
add multiple tables, and use primary and foreign key to query

### PR DESCRIPTION
**Pull request by joyapisi**
##  What I Did:
- Create a table named owners and populate it with data 
1A

- Create a table named species and populate it with data
1B

- Modify animals table:
  - Make sure that id is set as autoincremented PRIMARY KEY
  - Remove column species
  - Add column species_id which is a foreign key referencing species table
  - Add column owner_id which is a foreign key referencing the owners table
1C

- Insert data into the owners table:
1D

- Insert data into the species table:
1E

- Modify data inserted to animals table so it includes the species_id value:
  - If the name ends in "mon" it will be Digimon
  - All other animals are Pokemon
1F

- Modify your inserted animals to include owner information (owner_id):
1G

- Write queries (using JOIN) to answer the following questions:
  - What animals belong to Melody Pond?
AA

  -  List of all animals that are pokemon (their type is Pokemon).
BB

  -  List all owners and their animals, remember to include those that don't own any animal.
cc

  - How many animals are there per species
DD

  - List all Digimon owned by Jennifer Orwell.
EE

  - List all animals owned by Dean Winchester that haven't tried to escape.
FF

  - Who owns the most animals?
GG

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -66,3 +66,47 @@ FROM animals
...
+FROM animals
+JOIN owners ON animals.owner_id = owners.id
+JOIN species ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+
+SELECT animals.name
+FROM animals
+JOIN owners ON animals.owner_id = owners.id
+JOIN species ON animals.species_id = species.id
+WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
```
Comment (alexander16108) [inline queries.sql:105]: please as requested add the screenshot of the following queries to your PR description

- What animals belong to Melody Pond?
- List of all animals that are pokemon (their type is Pokemon).
- List all owners and their animals, remember to include those that don't own any animal.
- How many animals are there per species
- List all Digimon owned by Jennifer Orwell.
- List all animals owned by Dean Winchester that haven't tried to escape.

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @joyapisi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ Descriptive Readme file
✔️ Good PR
✔️ passing linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joyapisi** [comment]
# CHANGES MADE: 
- Update pull request to include required query photos

**Review by amejid — APPROVED** [review]
Hi @joyapisi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Correct workflow :heavy_check_mark: 
- SQL organized in multiple files :heavy_check_mark: 
- Good use of JOIN :heavy_check_mark: 
- Query screenshots added to PR description :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**